tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
is this relevant or could we remove it?

Reply:
This is added automatically when executing `go fmt ./...` for formatting the code.

Reply:
This seems to be used by the go command when building as tag indicator. As I'm currently using the Go version 1.17, extracting from the docs https://go.dev/doc/go1.17 , it says:

//go:build lines

> The go command now understands //go:build lines and prefers them over // +build lines. The new syntax uses boolean expressions, just like Go, and should be less error-prone. As of this release, the new syntax is fully supported, and all Go files should be updated to have both forms with the same meaning. To aid in migration, [gofmt](https://go.dev/doc/go1.17#gofmt) now automatically synchronizes the two forms. For more details on the syntax and migration plan, see [https://golang.org/design/draft-gobuild](https://go.dev/design/draft-gobuild).

In the same file we have also the tag `// +build tools` that should help `gofmt` in order to format the code when is run. This update seems better to be in another PR.